docs/ref/models/indexes.txt

```diff
@@ -32,22 +32,41 @@ options`_.
 
 .. attribute:: Index.fields
 
-A list of the name of the fields on which the index is desired.
+A list of the name of the fields or database expressions on which the index is
+desired.
 
 By default, indexes are created with an ascending order for each column. To
 define an index with a descending order for a column, add a hyphen before the
-field's name.
+field's name. When using an expression, use ``.desc()`` for descending order.
 
 For example ``Index(fields=['headline', '-pub_date'])`` would create SQL with
 ``(headline, pub_date DESC)``. Index ordering isn't supported on MySQL. In that
 case, a descending index is created as a normal index.
 
+``Index(fields=[Lower('title').desc()])`` would create an index on the
+lowercase value of the ``title`` field in descending order.
```

Comment:
Maybe it's also worth to add test and mention in doc that it's possible to create an index with math expressions, e.g.:
```python
Index(fields=[F('height')*F('weight')], name='mul_idx')
```